pagespeed/apache/mock_apache.cc

```diff
@@ -6,6 +6,10 @@
 #include "httpd.h"        // NOLINT
 #include "util_filter.h"  // NOLINT
 
+#include <cstdlib>
+#include <vector>
+#include "base/logging.h"
```

Comment:
This should probably go towards line 3 or so. Or is there some #define awkwardness that's getting in the way?